### DISCUSSION TITLE
Retrieve https image urls from cloudinary

### DISCUSSION THREAD

**Discussion by atinsley67** [tinacms/tinacms#4071] [Help]
Hello!

I have cloudinary configured as my media store in my nextjs/tina site, and my site (hosted on vercel) is available only over https (as is normal these days). However it doesn't look like there is a way to force the cloudinary media store to return image URLs as https - as a result I get a mixed content warning both in my browser development console and in the site quality checker (ahrefs).

Short of a post build script, are there any good options/workarounds for https for cloudinary images by default?

side note, I poked around in the timacms code and it looks like in the following (line 65) you only use the cloudinary API response 'url' as opposed to 'secure_url'. I suspect this means that tina won't support https cloudinary image URLs until you make some change here to support it.
  
https://github.com/tinacms/tinacms/blob/main/packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

I'm quite surprised this hasn't come up before (I did search for similar mentions) as it's very common to serve websites on https these days.
  
Thank you,
Alex

**Comment by jamespohalloran** [comment]
Good catch, I'm surprised this hasn't come up as well.
I just created an issue from this Discussion: https://github.com/tinacms/tinacms/issues/4072.
